scripts/setupdeps.sh

```diff
@@ -7,6 +7,11 @@ SUPPORTED_OS=('win64' 'maci64' 'maca64' 'glnxa64')
 DISTDIR="$(pwd)/dist/bin"
 mkdir -p $DISTDIR
 
+# Create plugins directory and copy buildtool plugins
+PLUGINDIR="$(pwd)/.matlab/plugins/+matlab/+ciplugins/+github"
+mkdir -p $PLUGINDIR
```

Comment:
I am not confident that this will work in general. I think it works for the CI for this repo because the `npm` tasks run as steps in the same job as we're running later so it is adding the plugins to `.matlab` in the local workspace. I think we want to put all of our "shipping code" somewhere in the dist folder. For example, `dist/` currently contains all of the generated javascript for our task (compiled from the typescript in our `src/`) along with any of the binaries we need for each plugin (run-matlab-command, genscript, etc.).

My suggestion is to copy the `plugins` folder into the dist folder so we have `dist/plugins/+matlab/+ciplugins/+github/*.m` and then we can add `dist/plugins` to path in the same way we add `dist/bin` to matlab path. You can see the `run-command` actions code for a good example of how to reference paths to files in the dist folder.

Reply:
Sound good thanks @davidbuzinski